libs/damap/src/lib/components/dmp/licenses/licenses.component.html

```diff
@@ -1,64 +1,73 @@
-</app-step-intro>
-<p>{{'dmp.steps.licensing.hint' | translate}}</p>
+<app-step-intro>{{ 'dmp.steps.licensing.intro' | translate }} </app-step-intro>
+<p>{{ 'dmp.steps.licensing.hint' | translate }}</p>
 <div [formGroup]="dmpForm">
   <ng-container formArrayName="datasets" *ngFor="let dataset of datasets?.controls; index as i">
     <mat-card *ngIf="dataset.value.source === datasetSource.NEW" class="mat-selected">
       <ng-container [formGroupName]="i">
         <div class="mat-card-container">
           <div class="mat-card-left">
-            <h4>{{dataset.value.title}}</h4>
-          </div>
-          <div class="mat-card-right">
-            <mat-form-field>
-              <mat-label>{{'dmp.steps.licensing.question.dataAccess' | translate}}</mat-label>
-              <mat-select formControlName="dataAccess">
-                <mat-option *ngFor="let type of accessType | keyvalue" [value]="type.key">
-                  {{translateEnumPrefix + type.value | translate}}<br/>
-                </mat-option>
-              </mat-select>
-            </mat-form-field>
+            <h4>{{ dataset.value.title }}</h4>
           </div>
         </div>
         <ng-container *ngIf="dataset.value.dataAccess === accessType.OPEN">
-          <div class="warning"
-               *ngIf="dataset.value.personalData || dataset.value.sensitiveData || dataset.value.legalRestrictions">
+          <div class="warning" *ngIf="
+              dataset.value.personalData ||
+              dataset.value.sensitiveData ||
+              dataset.value.legalRestrictions
+            ">
             <div class="warning-icon">
               <mat-icon>error_outline</mat-icon>
             </div>
             <ng-container *ngIf="dataset.value.personalData || dataset.value.sensitiveData">
-              {{'dmp.steps.licensing.warning.personalData' | translate}}
+              {{ 'dmp.steps.licensing.warning.personalData' | translate }}
               <ng-container *ngIf="!isAnonymisedOrPseudonymised">
-                {{'dmp.steps.licensing.warning.pseudonymisation' | translate}}
+                {{ 'dmp.steps.licensing.warning.pseudonymisation' | translate }}
               </ng-container>
             </ng-container>
             <ng-container *ngIf="dataset.value.legalRestrictions">
-              {{'dmp.steps.licensing.warning.legalRestrictions' | translate}}
+              {{ 'dmp.steps.licensing.warning.legalRestrictions' | translate }}
             </ng-container>
           </div>
-          <div class="mat-card-container">
+          <div id="licensingContainer" class="mat-card-container">
             <mat-form-field class="full-width">
-              <mat-label>{{'dmp.steps.licensing.question.license' | translate}}</mat-label>
-              <mat-select formControlName="license">
+              <mat-label>{{
+                'dmp.steps.licensing.question.license' | translate
+                }}</mat-label>
+              <mat-select [(value)]="defaultLicense">
```

Comment:
Using it like this sets the same license for all datasets in the view, but doesn't actually set the value for the DMP object. Setting a default license best happens in the [FormService](https://github.com/tuwien-csd/damap-frontend/blob/next/libs/damap/src/lib/services/form.service.ts#L286). `formControlName` then references this field in the form object. 
In `dev` mode (`nx serve ...`) the values stored in the form should be logged to the console and be updated every time any value changes, that way you can easily check what value is set and whether changes are reflected correctly.